lib/Renderer/StringRenderer.js

```diff
@@ -2,6 +2,8 @@ import {Renderer} from './Renderer';
 import {ValueProxy} from '../Data/ValueProxy';
 import {escape, escapeAttribute} from '../utils';
 
+const VOID_ELEMENTS = ['area', 'base', 'br', 'col', 'command', 'embed', 'hr', 'img', 'input', 'link', 'meta', 'param', 'source']
```

Comment:
In malanka template `<ANYTAG> === <ANYTAG /> === <ANYTAG></ANYTAG>` is true. The VOID_ELEMENTS using during server side rendering to produce string. In this case you don't want closing `<br>`. Compile one to one is not neccessery in my opinion